### DISCUSSION TITLE
Lesz-e BevfeldData XSD?

### DISCUSSION THREAD

**Discussion by szalaierika** [nav-gov-hu/eVAT#39] [Kérdések és válaszok - Gépi interfész] [answered]
Kedves NAV Szakértők,

azt szeretném megkérdezni, hogy BevfeldData XML-hez terveztek-e kiadni egy XSD struktúrát?

Előre is köszönöm!

**Comment by NTCA-HM** [comment]
Kedves @szalaierika !

Az ÁNYK XSD publikus, a Nyomtatványkitöltő telepítőjében megtalálható, de kitesszük a githubra is.

**Reply by szalaierika** [reply]
Kedves @NTCA-HM !
Nagyon köszönöm!

**Comment by szalaierika** [comment]
Nagyszerű, várjuk! Köszönjük!

**Reply by renced42** [reply, marked as answer]
Kedves @szalaierika 

Az XSD kihelyezésre került a https://github.com/nav-gov-hu/eVAT/tree/master/src/schemas/hu/gov/nav/abev alá.
Ezt egyébként a felhasználó gépére telepített ÁNYK program  is tartalmazza az ./abevjava/xsd mappa alatt.

**Comment by Macskafarka** [comment]
A link nem működik (404) sem Firefox, sem Google Chrome alatt és nincs a c:\users\felhasználó\abevjava alatt xsd mappa. sem :(

**Reply by renced42** [reply]
A linket javítottam, Github megtréfált.
C:/Users/Public/abevjava alatt keresed?
Ha az ÁNYK indító ikonjára kattintasz akkor ott látszik, hova telepítette illetve honnan indul.
Ott kell keresni.

**Comment by Macskafarka** [comment]
Link most már működik, és c:\users\Public\abevjava  alatt is ott van. Köszönöm a gyors segítséget!
